PathPlanning/RRTStar/rrt_star.py

```diff
@@ -141,6 +141,9 @@ def search_best_goal_node(self):
     def find_near_nodes(self, new_node):
         nnode = len(self.node_list) + 1
         r = self.connect_circle_dist * math.sqrt((math.log(nnode) / nnode))
+        # if expand_dist exists, search vertices in a range no more than expand_dist
+        if hasattr(self, 'expand_dis'): 
+            r = min(r, self.expand_dis)
```

Comment:
Is it from original rrt star paper? what is it for?

Reply:
Yes, this is from the paper. If you specified expanding distance, you must search the near neighbors in that range, so that neighbors are actually accessible from the new sample. 

But if the expanding distance does not exist, like LQR rrt*. You just search in the specified circle, since it's possible to steer the new sample to neighbors in that range.